src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/ExtractToCodeBehindCodeActionResolver.cs

```diff
@@ -148,7 +149,7 @@ private string GenerateCodeBehindPath(string path)
             string codeBehindPath;
             do
             {
-                var identifier = n > 0 ? n.ToString() : string.Empty;  // Make it look nice
+                var identifier = n > 0 ? n.ToString(CultureInfo.InvariantCulture) : string.Empty;  // Make it look nice
```

Comment:
It isn't clear to me that we'd want culture-specific numbers in a path

Reply:
I don't think I understand the question.

Reply:
Bet I misread your change, never mind❕